packages/manager/modules/pci/src/projects/project/sidebar/sidebar.constant.js

```diff
@@ -88,12 +88,19 @@ export const MENU = [
         title: 'Managed Kubernetes Service',
       },
       {
+        beta: true,
```

Comment:
Is this related to auto-backup ?

Reply:
no, its for private registry, its added because of rebase